src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx

```diff
@@ -0,0 +1,66 @@
+export const meta = {
+  title: 'Monitoring with Playwright on Checkly made easy',
+  description: "Use Playwright to run your status checks in the cloud to ensure the high availability of your web applications.",
+  author: "Max Schmitt",
+  date: "2020-09-23",
+  tags: ["checkly", "monitoring", "playwright"],
+  hidden: true
+}
+
+## Introduction
+
+Having a safe, functional, and feature-rich configured monitoring and status checks in place is nowadays needed to ensure a fast error reporting and by that a good uptime so your SLA is getting respected. The typical monitoring solutions are e.g. Grafana which would be feature-rich but has no good native integration for launching browsers to emulate a real user. Also, you need to host it yourself which is mostly for monitoring solutions not advised (or you need a HA installation in another data center or cloud provider).
+
+To fix these issues, Checkly has created a product to provide developer-friendly status checks, which can be used to launch browsers in the cloud. By using it you also don't need to spin up an own instance in your infrastructure since Checkly is a SaaS (Software as a Service) product and they host it for you. They even provide a full public dashboard for your customers. Starting from today they provide a full Playwright integration.
```

Comment:
Suggestion: You could further explain what the dashboards do, i.e. show end-users what the status of the service is (through Checkly's check results)